x-pack/test/common/services/index.ts

```diff
@@ -5,11 +5,13 @@
  */
 
 import { services as kibanaCommonServices } from '../../../../test/common/services';
+import { services as kibanaApiIntegrationServices } from '../../../../test/api_integration/services';
 
 import { SpacesServiceProvider } from './spaces';
 
 export const services = {
   ...kibanaCommonServices,
+  supertest: kibanaApiIntegrationServices.supertest,
```

Comment:
it was a master merge. i just moved this file out of `test/api_integrations/services`

Reply:
to clarify, i needed to add this import since the ingest manager service needs supertest after the merge from master. I moved the file from  `api_integration/services/ingest_manager` to `common/services/ingest_manager` since api_integration is using ingest and so is security_solution_endpoint. i could also just add that import directly to the ingest_manager service file, if you think thats better

Reply:
Thank you - now I understand. I guess for Endpoint UI Functional, we did not have to do that because we import the services from `x-pack/test/functional/` which (my guess) is already brings that service in.
Anyway 🙏